src/v/raft/types.h

```diff
@@ -460,6 +460,16 @@ struct timeout_now_reply {
     status result;
 };
 
+struct transfer_leadership_request {
+    group_id group;
```

Comment:
offline sync on this: this shouldn't affect correctness. if a wild request hit the wrong instance of a partition (ie a vnode) then it would still either be the current leader or not, and that is the only checking that is important (unlike the extra checking needed for heartbeats / append_entries).